certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -699,6 +724,7 @@ PROCEDURE READCONFIGFILESETTINGS
 **  TT             PARAMXFERTOSHARES       Valid share types to transfer funds from cert to share (in LISTEXPAND format)
```

Comment:
I recommend changing the comment for ** ET to mention that it is for share transfer type 2 records.

Reply:
Will do